src/motion/context/MotionContext.ts

```diff
@@ -121,6 +122,7 @@ export const useMotionContext = (
             animate: targetAnimate,
             values,
             hasMounted,
+            isReducedMotion: parentContext.isReducedMotion,
```

Comment:
will this propagate `isReducedMotion` change correctly if it changes? 🤔 it's not used as a dependency to this memo